.env

```diff
@@ -0,0 +1 @@
+JOURNAL_API_URL=https://n7y4jadh08.execute-api.eu-west-1.amazonaws.com/alpha/email
```

Comment:
don't check your .env file in, should be in .gitignore

Reply:
*secret repo* should let you commit it? I thought that it will catch it